test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
I'm always in favor of instance methods over class-level.  The Sinatra/Rack API is bad in that regard.  I'd much prefer to see something like:

```ruby
web = Sidekiq::Web.new
web.something = ...
run web
```